modules/network/nsgs.tf

```diff
@@ -364,6 +364,30 @@ resource "oci_core_network_security_group_security_rule" "int_lb_egress" {
   count = var.load_balancers == "internal" || var.load_balancers == "both" ? length(local.int_lb_egress) : 0
 }
 
+resource "oci_core_network_security_group_security_rule" "int_lb_ingress" {
+  network_security_group_id = oci_core_network_security_group.int_lb[0].id
+  description               = "Allow stateful ingress from ${element(element(local.internal_lb_allowed_cidrs_and_ports, count.index), 0)} on port ${element(element(local.internal_lb_allowed_cidrs_and_ports, count.index), 1)}"

```

Comment:
The description generated is not correct . When we provide internal_lb_allowed_cidrs and internal_lb_allowed_ports the description is not correct. please review.

Reply:
The issue is because of lifecycle ignore_changes which are messing up the rules. We added those because 1) we were using security list 2) ccm was adding nodeports 3) running terraform would remove the rules added by ccm. 

Since we are not using security list for load balancer and workers anymore, let's investigate removing the lifecyle ignore_changes on the NSGs